### DISCUSSION TITLE
Agent-Linux and Windows

### DISCUSSION THREAD

**Discussion by Ronnyhe96** [Ylianst/MeshCentral#5676] [General]
What could it be if an agent is already installed on ubuntu and windows but sometimes it takes a long time to see it on my panel
![Captura de pantalla 2023-07-18 112815](https://github.com/Ylianst/MeshCentral/assets/135432623/d483da2c-e008-47c0-aee4-6362eda75319)

**Comment by si458** [comment]
it depends on your config.json, i know the is an issue at the moment, where if you have sessionrecording enabled, 
you can add machines for a period of time, then suddenly, you cant add more devices? but previously added devices work no problem?
